### DISCUSSION TITLE
添加种子文件下载功能

### DISCUSSION THREAD

**Discussion by xxtyyq** [jxxghp/MoviePilot#3464] [讨论]
### 当前程序版本

1.7.9

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

希望有和NT一样的直接添加种子文件进行下载的功能，一直没有找到，不确定是不是没有？

**Comment by zhuxiongtao** [comment]
这个功能我也找了半天没找到应该是没有吧，有时候下载些特定资源很不方便

**Comment by dlovew** [comment]
有个插件，仔细看看

**Comment by sunnyvGitHub** [comment]
> 有个插件，仔细看看

请问插件名字是什么？没找到相关插件

**Comment by JaylenMar** [comment]
<img width="641" alt="截屏2024-05-12 00 35 35" src="https://github.com/jxxghp/MoviePilot/assets/66826612/388530da-aecb-493f-8468-7d872b2124a7">

**Comment by xxtyyq** [comment]
是不是其他插件库的？确实没找到
